akka-actor/src/main/scala/akka/actor/Deployer.scala

```diff
@@ -41,14 +54,26 @@ object Deploy {
  * }}}
  */
 @SerialVersionUID(3L)
-final case class Deploy(
-    path: String = "",
-    config: Config = ConfigFactory.empty,
-    routerConfig: RouterConfig = NoRouter,
-    scope: Scope = NoScopeGiven,
-    dispatcher: String = Deploy.NoDispatcherGiven,
-    mailbox: String = Deploy.NoMailboxGiven,
-    tags: Set[String] = Set.empty) {
+final class Deploy(
```

Comment:
do we have a protobuf serializer for Deploy that needs adjustment to include the tags?

Reply:
Yes, that needs to be done before this is completed (in progress with that)